data/ecosystems/h/hedera.toml

```diff
@@ -32,13 +32,20 @@ github_organizations = [
   "https://github.com/hashgraph",
   "https://github.com/Hashpack",
   "https://github.com/hedera-dev",
+  "https://github.com/iassetsorg",
```

Comment:
this is github user URL and not a github org URL, can you remove and we keep the repos you added only?

Reply:
done
Thanks